script.js

```diff
@@ -0,0 +1,17 @@
+var GalaxyS6 = new Telefon('Samsung', 2200, 'czarny');
+var iPhone6S = new Telefon('Apple', 3300, 'srebrny');
+var OnePlusOne = new Telefon('OnePlus', 1100, 'bialy');
+
+function Telefon(marka, cena, kolor) {
+	this.marka = marka;
+	this.cena = cena + ' PLN';
```

Comment:
zamiast dawać `PLN` na sztywno to już lepiej by było utworzyć nowe property `this.currency = currency || 'PLN'` - wtedy skrypt będzie miał szersze zastosowanie